src/BrowsershotLambda.php

```diff
@@ -54,4 +56,32 @@ protected function throwError(SettledResult $response): void
 
         $response->throw();
     }
+
+    /**
+     * @throws CouldNotTakeBrowsershot
+     */
+    public function saveToS3(string $targetPath, string $disk = 's3')
+    {
+        $this->setOption('s3', [
+            'path' => $targetPath,
+            'region' => config('sidecar.region'),
+            'bucket' => config("filesystems.disks.$disk.bucket")
```

Comment:
```suggestion
            'bucket' => config('filesystems.disks.$disk.bucket')
```

Reply:
I don't think this will work. Doesn't it need to be double quotes for the variable to be interpolated?

Reply:
Yeah. Totally right. Missed that.